infra/k8s/cluster.yaml

```diff
@@ -12,6 +12,9 @@ spec:
   channel: stable
   cloudProvider: aws
   configBase: ${KOPS_STATE_STORE}/${NAME}
+  docker:
+    defaultUlimit:
+    - "nofile=1048576:1048576"
```

Comment:
@coryschwartz maybe use a different value, just so that we can confirm we can actually change it.

Currently on previous clusters we have:
```
➜  testground git:(disable-pushgateway) k exec -it dummy-tt89z sh
/ # ulimit -a
-f: file size (blocks)             unlimited
-t: cpu time (seconds)             unlimited
-d: data seg size (kb)             unlimited
-s: stack size (kb)                8192
-c: core file size (blocks)        unlimited
-m: resident set size (kb)         unlimited
-l: locked memory (kb)             64
-p: processes                      1048576
-n: file descriptors               1048576
-v: address space (kb)             unlimited
-w: locks                          unlimited
-e: scheduling priority            0
-r: real-time priority             0
/ #
```